### ISSUE TITLE
P-value bar is detached from the volcano plot

### ISSUE THREAD

**Issue by tj8901nm** [greenelab/adage-server#135]
![image](https://cloud.githubusercontent.com/assets/13037102/25627053/6bef1afc-2f2f-11e7-9d40-617a82495a14.png)

**Comment by mhuyck** [comment]
Whoa! Thanks for the excellent screen grab. I'll get this sorted out.

**Comment by mhuyck** [comment]
This only happens for some plots... @tj8901nm, can you include a list of the samples you're comparing to make it easier for me to reproduce? You could just take another screen shot of the contents of your "sample bin" (the icon with test tubes and a number in the nav bar at the upper right).

**Comment by tj8901nm** [comment]
Yes, I found that this only happens when I tried to return to the sample page and then plot the volcano plot again. Here are the samples I used.

![image](https://cloud.githubusercontent.com/assets/13037102/25630633/cd61db2e-2f3b-11e7-9575-c0d759751f51.png)

**Comment by mhuyck** [comment]
Thank you!

**Comment by mhuyck** [comment]
I think this is an unintended consequence of the way I added the p-value bar to the plot.

**Comment by tj8901nm** [comment]
I realized that the p-value bar did not even exist when I first try to generate the volcano plot for these samples.

**Comment by mhuyck** [comment]
Gotcha. I'm able to reproduce it using your example "Anr" search. Thank you!

**Comment by mhuyck** [comment]
The vega plot really shouldn't even be able to get that tall. Maybe a vega bug, but possible there will be a way for me to work around it. There is also a new version of vega now (v3.0 beta), so we will need to port our work over to the new spec format sooner or later.

**Comment by mhuyck** [comment]
Guide for porting from Vega 2 to 3:

https://vega.github.io/vega/docs/porting-guide/
